fedrec/modules/embeddings.py

```diff
@@ -137,6 +215,26 @@ def __init__(self,
                     torch.tensor(W, requires_grad=True))
 
     def forward(self, input, offsets=None, per_sample_weights=None):
+        """
+        Forward pass of EmbeddingBag.
+
+        Arguments
+        --------------
+        input (Tensor)  Tensor containing bags of indices into the embedding matrix.
```

Comment:
```suggestion
        input(Tensor)- Tensor containing bags of indices into the embedding matrix.
```
Please write the arguments/parameters in this way across the file.